src/io/ctrees_utils.c

```diff
@@ -673,7 +680,7 @@ int64_t find_fof_halo(const int64_t totnhalos, const struct additional_info *inf
         return start_loc;/* should never be called to find the FOF of a FOF halo -> but whatever*/
     }
 
-    const int64_t max_recursion_depth = 30, recursion_depth_for_verbose = 3;
+    const int64_t max_recursion_depth = 30, recursion_depth_for_verbose = 5;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max_recursion_depth' is assigned a value that is never used.](https://app.codacy.com/app/jacobseiler/sage-model/pullRequest?prid=4171404)